internal/statesync/snapshots.go

```diff
@@ -126,6 +127,19 @@ func (p *snapshotPool) Best() *snapshot {
 	return ranked[0]
 }
 
+// At returns the known snapshot at the height.
+// If there are multiple snapshots or no snapshot at the height, it will return nil.
+func (p *snapshotPool) At(height uint64) (*snapshot, error) {
+	keys := p.heightIndex[height]
+	if len(keys) > 1 {
+		return nil, errors.New("multiple snapshots at height")
```

Comment:
from the above comment: "If there are multiple snapshots or no snapshot at the height, it will return nil."

please amend the comment or the code